src/library/scala/Option.scala

```diff
@@ -313,6 +442,14 @@ sealed abstract class Option[+A] extends Product with Serializable {
 
   /** Returns a singleton list containing the $option's value
    * if it is nonempty, or the empty list if the $option is empty.
+   *
+   * This is equivalent to:
+   * {{{
+   * option match {
+   *   case Some(x) => List(x)
+   *   case None    => Nil
+   * }
+   * }}}
    */
   def toList: List[A] =
     if (isEmpty) List() else new ::(this.get, Nil)
```

Comment:
not part of this PR, but the fact that it uses `List()` and not `Nil` seems weird.

Reply:
Indeed, there are a dozen of them throughout the library.